modules/loki-rate-limit-errors.adoc

```diff
@@ -0,0 +1,83 @@
+// Module is included in the following assemblies:
+// * logging/cluster-logging-loki.adoc
+// * logging/cluster-logging-external.adoc
+
+:_content-type: PROCEDURE
+[id="loki-rate-limit-errors_{context}"]
+= Troubleshooting Loki rate limit errors
+
+If the Log Forwarder API forwards a large block of messages to Loki that exceeds the rate limit, Loki generates rate limit (`429`) errors.
+
+These errors can occur during normal operation. For example, when adding {ocp-logging} to a cluster that already has some logs, rate limit errors might occur while {ocp-logging} tries to ingest all of the existing log entries. In this case, if the rate of addition of new logs is less than the total rate limit, the historical data is eventually ingested, and the rate limit errors are resolved without requiring user intervention.
+
+In cases where the rate limit errors do not resolve themselves, you can fix the issue by modifying the `LokiStack` custom resource (CR).
+
+[IMPORTANT]
+====
+The `LokiStack` CR is not available on Grafana-hosted Loki. This topic does not apply to Grafana-hosted Loki servers.
+====
+
+.Conditions
```

Comment:
Is this a standard subheading? I have not seen it before

Reply:
No, but it was preexisting in the docs, so I left it as is.